lib/providers/player/player_provider.dart

```diff
@@ -14,100 +14,99 @@ final playerProvider =
 class PlayerProvider extends StateNotifier<TrackModel?> {
   PlayerProvider(this.ref) : super(null);
   Ref ref;
+
   Future<void> loadSelectedTrack({
     required TrackModel trackModel,
     required TrackFilesModel file,
   }) async {
-    var _track = trackModel.customCopyWith();
-    for (var i = 0; i < _track.audio.length; i++) {
-      var element = _track.audio[i];
-      var fileIndex = element.files.indexWhere((e) => e.id == file.id);
+    var track = trackModel.customCopyWith();
+
+    track.audio.forEach((audioModel) {
+      var fileIndex = audioModel.files.indexWhere((it) => it.id == file.id);
```

Comment:
what's the meaning of **it**?

Reply:
It's the same as a for loop, but directly gives you the item instead if incrementing an index